accounts/static/accounts/js/onboarding_movies.js

```diff
@@ -0,0 +1,522 @@
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error("Network response was not ok");
+        }
+
+        return response.json();
+      })
+      .then((data) => {
+        movieContainer.innerHTML = "";
+        data.movies.forEach((movie) => {
+          var movieDiv = createMovieDiv(movie);
+          movieContainer.appendChild(movieDiv);
+        });
+      });
+  } else {
+    movieContainer.innerHTML = ""; //TODO create new div to add response message ("Getting random movies...")
+    fetchMovies(50);
+  }
+}
+
+/**
+ * Fetches random movies from the database and adds them to the movie container.
+ * @param {int} numMovies the number of movies to fetch
+ * @returns
+ */
+async function fetchMovies(numMovies = 35) {
+  // Prevent multiple fetches at once
+  if (isLoading) return;
+  isLoading = true;
+
+  try {
+    // Fetch movies from the API
+    const response = await fetch(`/api/random/movies/?amount=${numMovies}`);
+    if (!response.ok) {
+      throw new Error("Network response was not ok");
+    }
+    // Load JSON
+    const data = await response.json();
+    data.movies.forEach((movie) => {
+      var movieDiv = createMovieDiv(movie);
+      movieContainer.appendChild(movieDiv);
+    });
+  } catch (error) {
+    console.error("Error fetching movies:", error);
+  } finally {
+    isLoading = false;
+  }
+}
+
+/**
+ * This function is called when the user clicks the submit button on the onboarding page,
+ * if the response is successful, the user is redirected to the triggers page.
+ */
+async function submitOnboardingMovieForm() {
+  const csrfToken = await getCSRFToken();
+  const data = {
+    movies_liked: Array.from(movies_liked),
+    movies_disliked: Array.from(movies_disliked),
+    movies_watched: Array.from(movies_watched),
+    watchlist: Array.from(watchlist),
+    movies_rewatch: Array.from(movies_rewatch),
+    movies_blocked: Array.from(movies_blocked),
+  };
+
+  fetch("/accounts/onboarding/movies/", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "X-CSRFToken": csrfToken,
+    },
+    body: JSON.stringify(data),
+  })
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      }
+
+      window.location.href = "/accounts/onboarding/triggers/";
+    })
+    .catch((error) => {
+      console.error("Error submitting movie preferences:", error);
+    });
+}
+
+function redirectToOnboardingGenreForm() {
+  window.location.href = "/accounts/onboarding/genres/";
+}
+
+function openBackModal() {
+  document.getElementById("confirm_back_modal").classList.remove("hidden");
+  document.getElementById("exit_overlay").classList.remove("hidden");
+}
+
+function closeBackModal() {
+  document.getElementById("confirm_back_modal").classList.add("hidden");
+  document.getElementById("exit_overlay").classList.add("hidden");
+}
+
+/************************************
+ *
+ *  Page initialization
```

Comment:
dear god

and then why is this separator a different format than the other!? AND WHY IS IT FIVE LINES LONG